### DISCUSSION TITLE
Series Management Feature

### DISCUSSION THREAD

**Discussion by erinb223** [forem/forem#19344] [Feature Requests]
We got some feedback on Sloan's post about series creation:

https://dev.to/kamil7x/comment/265ch
> "Unfortunately, if you mistype series name, it will still be created, and you will have additional item inside Existing series dropdown. I hope there will be possibility for managing series in future."

This is a great point, and I wanted to get it on our radar. I think a series management feature would give content creators and pro tools users more agency as they curate their content.

**Comment by kamil7x** [comment]
Actually I opened a discussion about it some time ago
https://github.com/forem/forem/discussions/16434

I can see there are also others:
https://github.com/forem/forem/discussions/15673

**Comment by erinb223** [comment]
Closing this discussion to roll into [#16434](https://github.com/forem/forem/discussions/16434).
